thriftserver/thriftserver/base/thriftserver-secret.yaml

```diff
@@ -0,0 +1,8 @@
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: thriftserver-secret
+stringData:
+  s3_key_id: SECRET_KEY_ID
+  s3_access_key: SECRET_ACCESS_KEY
```

Comment:
Theses two lines should be parametrized

Reply:
I don't agree with that, see below.

Reply:
resolved via `params.env` templating a secret name in deployments

```
s3_credentials_secret=thriftserver-sample-s3
```